rpc_jobs/rpc_upgrades.yml

```diff
@@ -63,19 +82,18 @@
       - "kilo_to_r14.4.1_leap"
     jira_project_key: "RLM"
     jobs:
-      - 'PR_{repo_name}-{series}-{image}-{scenario}-{action}'
       - 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
 
 - project:
-    name: "rpc-upgrades-mnaio-newton-head"
+    name: "rpc-upgrades-mnaio-newton-head-pm"
     repo_name: "rpc-upgrades"
     repo_url: "http://github.com/rcbops/rpc-upgrades"
     branches:
      - "master"
     image:
-      - mnaio:
+      - trusty_mnaio:
           FLAVOR: "onmetal-io1"
-          IMAGE: "OnMetal - Ubuntu 14.04 LTS (Trusty Tahr)"
+          IMAGE: "OnMetal - Ubuntu 16.04 LTS (Xenial Xerus)"
```

Comment:
It's probably worth adding a comment here to explain why the IMAGE value does not match the image axis name, otherwise both reviewers and later job changers are going to be very confused.